### ISSUE TITLE
commit #7e61b95 has broken build

### ISSUE THREAD

**Issue by k0d3r1s** [Imagick/imagick#595]
Hi. As far as my non-existent c knowledge goes, i think that this comment to develop has broken build.  
This commit (https://github.com/Imagick/imagick/commit/7e61b9523dfb10a97feebbdcd7f98924212d8d05) added:  
```c
#if MagickLibVersion >= 0x711
    IMAGICK_REGISTER_CONST_LONG("COMPRESSION_BC5", BC5Compression);
    IMAGICK_REGISTER_CONST_LONG("COMPRESSION_BC7", BC7Compression);
#endif
```
but few lines below this, one of these constants is already defined:  
```c
#if MagickLibVersion >= 0x711
	// Technically >= 7.1.0-13 but we still don't have a mechanism for
	// detecting patch versions.
	IMAGICK_REGISTER_CONST_LONG("COMPRESSION_BC7", BC7Compression);
#endif
```
so, now building this extension from develop branch fails with error about redefining already defined constant.  
if i'm wrong, sorry, but I think this a bug needed to be fixed

**Comment by Danack** [comment]
HI @k0d3r1s 

To set your expectations, the only branch I try hard to keep the CI green on is the master branch. The develop branch sometimes has some commits in it that indeed might fail, due to various reasons.

And to be clear, the only fully supported versions are those that are released through PECL.

I had already fixed the problem in a branch in 72428f83ce5e825149ecd0255ba9e737eca2b072 in a different branch, but hadn't merged it, but it is now.

If you do want to use something from the develop branch, I'd suggest pinning to a particular commit SHA, rather than getting latest. But using the master branch would be safer, but not guaranteed, from having issues.

**Comment by k0d3r1s** [comment]
Thanks for reply (and fix) :) I'm using php built from master (8.3) and last time i tried to build from imagick master, there were some things that did not build properly but does build from develop.  
anyway, thanks and i will check if i can use master onwards
